### ISSUE TITLE
customize store and single product page

### ISSUE THREAD

**Issue by AteS-Github** [bootscore/bootscore#135]
hello woocommerce i couldn't change product and rows per row
actually I want to change the product views, can you help me?

[https://kombiservisklima.com/magaza/](url)

[https://kombiservisklima.com/wp-content/uploads/2022/04/Ekran-Alintisi.png](url)

**Comment by crftwrk** [comment]
It's not possible to change products in row by the Customizer. This must be done in template file. 

- Copy `/woocommerce/content-product.php` to your child. 
- In line `29` you will find `col-md-6 col-lg-4 col-xxl-3` classes.
- Change column classes as you want https://getbootstrap.com/docs/5.1/layout/grid/#grid-options

Does this help?

**Comment by AteS-Github** [comment]
thank you for the quick response
I fiddled with the codes a bit, but I didn't get the look I wanted.
For example, let's show 4 products in 1 row on my store page.
How do I get it to show a total of 4 lines?

**Comment by crftwrk** [comment]
Please have a look to Bootstrap Grid docs again https://getbootstrap.com/docs/5.1/layout/grid/.

To show 4 products in a row do this:

- Open `/woocommerce/content-product.php` and go to line `29`
- Change classes `col-md-6 col-lg-4 col-xxl-3` to `col-3 col-md-3 col-lg-3 col-xxl-3`

Now you have 4 items per row on every screen size. The grid is a 12-column layout. 12 divided by 3 is 4. Using `col-6 col-md-6 col-lg-3 col-xxl-3` classes for example means 2 columns mobile / tablet, 4 columns .desktop

**Comment by AteS-Github** [comment]
I did it, no problem on the desktop, it looks like I want, but it breaks the site on mobile, so I couldn't do it.

For example, let 4 products appear in 1 row on the table.
How do we set 2 products in 1 line to be 4 lines in total on mobile?

**Comment by AteS-Github** [comment]
With this code, the view is ok, but for example, it does not auto-adjust and shorten the product titles.

col-6 col-md-6 col-lg-3 col-xxl-3

https://herbalicin.com/magaza/

**Comment by crftwrk** [comment]
> it does not auto-adjust and shorten the product titles.

Of course, because this is a `<h2>`. You must adjust font-size by yourself with CSS. For example:

```css
h2.woocommerce-loop-product__title {
  font-size: calc(1.275rem + .3vw);
}
```

**Comment by AteS-Github** [comment]
I CAN'T FIND THIS IN THE THEME
h2 . woocommerce-loop-product__title {
   font-size :  calc ( 1.275 rem  +  .3 vw );
}

**Comment by crftwrk** [comment]
> I CAN'T FIND THIS IN THE THEME

This is not part of the theme. You must add this snippet in in your child `scss/_bscore_custom.scss`.

**Comment by AteS-Github** [comment]
:(

**Comment by crftwrk** [comment]
bootScore is a developer theme. You must know the basics of Bootstrap, HTML and CSS to customise it.

**Comment by AteS-Github** [comment]
/*!
 * bootScore custom styles
 */

// Add your own code here!
// Because this file is compiled after Bootstrap, you're also able to use Bootstrap mixins for better code.
// Some examples can be found here: https://bootscore.me/documentation/bootstrap-css-sass

h4 . woocommerce-loop-product__title {
   font-size :  calc ( 1.275 rem  +  .3 vw );
}

I'M LEARNING HERE

THIS WAY I ADDED NO CHANGES

**Comment by crftwrk** [comment]
Perfect. But your code will not work, because its `h2` and not `h4`. Use this:

```css
h2.woocommerce-loop-product__title {
  font-size: calc(1.275rem + .3vw);
}

```

**Comment by AteS-Github** [comment]
h2 . woocommerce-loop-product__title {
   font-size :  calc ( 1.275 rem  +  .3 vw ); 
}

I added this but it didn't work?

**Comment by crftwrk** [comment]
Yes, it works. Your font-size is smaller. Adjust size as you want:

```css
h2.woocommerce-loop-product__title {
  font-size: 11px;
}
```

**Comment by AteS-Github** [comment]
yes, that's it, I'll find out in time, of course, if you don't mind dealing with me, thank you for your help
